django/contrib/auth/views.py

```diff
@@ -200,6 +201,10 @@ def password_reset(request, is_admin_site=False,
                     RemovedInDjango110Warning, 3
                 )
                 opts = dict(opts, domain_override=request.get_host())
+
+            if extra_email_context is not None:
+                opts.update({'extra_email_context': extra_email_context})
```

Comment:
I see. It looks like I commented in the wrong place. I don't think we need the `is not None` check here -- we can put the key/value in the `opts` dict in the view. In `forms.py` that's where it should be okay to use `context.update( extra_email_context)`.